prov/efa/src/rdm/rxr_op_entry.c

```diff
@@ -615,18 +612,15 @@ void rxr_rx_entry_handle_error(struct rxr_op_entry *rx_entry, int err, int prov_
 	err_entry.buf = rx_entry->cq_entry.buf;
 	err_entry.data = rx_entry->cq_entry.data;
 	err_entry.tag = rx_entry->cq_entry.tag;
+	if (OFI_UNLIKELY(efa_rdm_error_write_msg(ep, rx_entry->addr, err, prov_errno,
+											 &err_entry.err_data, &err_entry.err_data_size) < 0)) {
+		err_entry.err_data_size = 0;
+	}
```

Comment:
nit: indent

functional: I think efa_rdm_error_write_msg may leave err_entry.err_data un-initialized on error.  I can't tell from github if err_entry.err_data is set to NULL ahead of time.